samples/lightdb_led/README.rst

```diff
@@ -128,21 +128,27 @@ sample application (i.e., ``samples/lightdb_led``) and type:
    $ west build -b nrf52840dk_nrf52840 samples/lightdb_led
    $ west flash
 
-nRF9160 Feather
----------------
+nRF9160-based devices
+---------------------
 
 On your host computer open a terminal window, locate the source code of this
 sample application (i.e., ``samples/lightdb_led``) and type:
 
 .. code-block:: console
 
-   $ west build -b circuitdojo_feather_nrf9160_ns samples/lightdb_led
+   $ #Build for the Circuit Dojo nrf9160 Feather:
+   $ west build -b circuitdojo_feather_nrf9160_ns samples/lightdb_led -p
```

Comment:
Could you please squash `-p` removal? There is no point in introducing it and removing right away in subsequent commit in the same PR.

Reply:
Done! Please take a look when you have a chance. Thanks!